### ISSUE TITLE
500 error on hiring

### ISSUE THREAD

**Issue by g4s8** [zerocracy/farm#1045]
0crat failed to send hiring text, it deducted 128 reputation but didn't send the text with error:
```
java.lang.IllegalStateException: [POST https://www.0crat.com/hiring-send/CAWJH9K0X] failed in 752ms: File "vacancies.xml" is not accessible in "PMO"
	at org.takes.facets.fallback.TkFallback.error(TkFallback.java:234)
	at org.takes.facets.fallback.TkFallback.route(TkFallback.java:115)
	at org.takes.facets.fallback.TkFallback.access$000(TkFallback.java:54)
	at org.takes.facets.fallback.TkFallback$1.act(TkFallback.java:66)
	at org.takes.tk.TkWrap.act(TkWrap.java:61)
	at org.takes.tk.TkSslOnly.act(TkSslOnly.java:71)
	at org.takes.tk.TkWrap.act(TkWrap.java:61)
	at org.takes.http.BkBasic.print(BkBasic.java:108)
	at org.takes.http.BkBasic.accept(BkBasic.java:84)
	at org.takes.http.BkSafe$1.accept(BkSafe.java:51)
	at org.takes.http.BkWrap.accept(BkWrap.java:52)
	at org.takes.http.BkTimeable.accept(BkTimeable.java:89)
	at org.takes.http.BkParallel$1$1.run(BkParallel.java:89)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.IllegalArgumentException: File "vacancies.xml" is not accessible in "PMO"
	at com.zerocracy.farm.strict.StrictProject.acq(StrictProject.java:112)
	at com.zerocracy.farm.props.PropsProject.acq(PropsProject.java:81)
	at com.zerocracy.farm.footprint.FtProject.acq(FtProject.java:67)
	at com.zerocracy.farm.ruled.RdProject.acq(RdProject.java:54)
	at com.zerocracy.farm.reactive.RvProject.acq(RvProject.java:62)
	at com.zerocracy.pmo.Pmo.acq(Pmo.java:58)
	at com.zerocracy.pmo.Vacancies.item(Vacancies.java:139)
	at com.zerocracy.pmo.Vacancies.bootstrap(Vacancies.java:69)
	at com.zerocracy.tk.project.TkHiring.act(TkHiring.java:76)
	at org.takes.facets.fork.FkRegex.route(FkRegex.java:210)
	at org.takes.facets.fork.FkChain.route(FkChain.java:75)
	at org.takes.facets.fork.TkFork.act(TkFork.java:100)
	at com.zerocracy.tk.TkSoftForward.lambda$new$0(TkSoftForward.java:50)
	at org.takes.tk.TkWrap.act(TkWrap.java:61)
	at org.takes.facets.forward.TkForward.act(TkForward.java:68)
	at org.takes.facets.fork.FkFixed$1.route(FkFixed.java:56)
	at org.takes.facets.fork.FkWrap.route(FkWrap.java:61)
	at org.takes.facets.fork.FkChain.route(FkChain.java:75)
	at org.takes.facets.fork.TkFork.act(TkFork.java:100)
	at org.takes.facets.auth.TkAuth.act(TkAuth.java:109)
	at org.takes.facets.auth.TkAuth.act(TkAuth.java:89)
	at org.takes.tk.TkWrap.act(TkWrap.java:61)
	at org.takes.facets.flash.TkFlash.act(TkFlash.java:108)
	at org.takes.tk.TkGzip$1.act(TkGzip.java:58)
	at org.takes.tk.TkWrap.act(TkWrap.java:61)
	at org.takes.tk.TkMeasured$1.act(TkMeasured.java:67)
	at org.takes.tk.TkWrap.act(TkWrap.java:61)
	at org.takes.tk.TkVersioned$1.act(TkVersioned.java:73)
	at org.takes.tk.TkWrap.act(TkWrap.java:61)
	at org.takes.tk.TkWithHeaders$1.act(TkWithHeaders.java:71)
	at org.takes.tk.TkWrap.act(TkWrap.java:61)
	at org.takes.facets.fallback.TkFallback.route(TkFallback.java:86)
	... 14 more
```

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by yegor256** [comment]
@g4s8 fixed in 19dff08d7904dfb694ea58c6a54aed20343cf188 thanks!
